Assets/Scripts/2D/GuiManagerScript.cs

```diff
@@ -2005,8 +2017,8 @@ public void PostProgressOp_LoadAction()
 
         _postProgressOp -= PostProgressOp_LoadAction;
 
-        if (_loadWorldPostProgressOp != null)
-            _loadWorldPostProgressOp.Invoke();
+        _loadWorldPostProgressOp?.Invoke();
```

Comment:
I like all the null propagation

Reply:
Thanks, I saw you using it and it's way better than adding if(null) blocks